libraries/FFGAppImport/AssetImport/AppFinder.cs

```diff
@@ -89,13 +91,11 @@ public AppFinder(Platform p)
             else if (platform == Platform.Android)
             {
                 obbRoot = Android.GetStorage() + "/Valkyrie/Obb";
+                ValkyrieDebug.Log("Obb extraction path: " + obbRoot);
                 location = obbRoot + "/assets/bin/Data";
-                if (Directory.Exists(obbRoot))
-                {
-                    Directory.Delete(obbRoot, true);
-                }
+                //DeleteObb();
```

Comment:
Won't this leave lots of data lying around?

Reply:
Actually this is on start.  This was here incase of a failed import/crash to clean up.  Unless there is a good reason I'd rather put it back in.

Reply:
Went into problems with that, it's called multiple times. Maybe because of: 
`if (importData.platform == Platform.Android) return new MoMFinder(Platform.Android).RequiredFFGVersion(); // TODO: implement version checking for Android` in `fetchAppVerison()`. Will test it tomorrow.